src/Mutation/PriorityNodeTraverser.php

```diff
@@ -33,7 +33,7 @@
 
 declare(strict_types=1);
 
-namespace Infection\Traverser;
+namespace Infection\Mutation;
```

Comment:
What's the reason of this change? 

Currently, this namespace contains `FileParser` and `UnparsableFile` classes. Adding Traverser there makes it even more "too common". At lease I don't understand what we are going to add to this namespace.

Reply:
The reason of this specific change is that having a `Traverser` namespace here does not make any sense to me at least right now.

There is a lot of debate on how to organise things, but unless you have a specific adapter in your applications, I feel the "keep things that change together close to each other" a good rule of thumb. So in `Infection\Mutation` it's primarily `MutationGenerator`. The parse, traverser, traverser factory are merely sub elements of it.

The main alternative I can think of that makes sense as well is having "PhpParser adapter", e.g. like [here in PHP-Scoper](https://github.com/humbug/php-scoper/tree/master/src/PhpParser). I think however that it won't suit Infection as well as I would really prefer to keep the mutators, which directly depend on PhpParser, under the `Mutator` directory.

Reply:
One additional note @maks-rafalko because I feel I failed to express that. I don't think my current changes will be the "final" way either. I however prefer this structure over a separation of things by type (e.g. putting every model under `Model`, every exception under `Exception`) because IMO it's makes it easier to see what's closely related to each other.

I still have quite _a lot_ of refactoring planned for the next months; after those I think it would be interesting to have a review as a team about the organisation to see if what we are not satisfied with and if we could improve it in some way.

Reply:
no problem, thanks for the information